packages/flame/lib/src/gestures/events.dart

```diff
@@ -26,6 +26,23 @@ class EventPosition {
   EventPosition(this._game, this._localPosition, this._globalPosition);
 }
 
+/// [EventDelta] converts deltas based events to two different values (game and global).
+///
+/// global: this is the raw value received by the event without any scale applied to it
+/// game: the scalled value applied all the game transformations
```

Comment:
```suggestion
/// [global]: this is the raw value received by the event without any scale applied to it; this is always the same as local because Flutter doesn't apply any scaling.
/// [game]: the scalled value applied all the game transformations.
```